docs/details/widgets/scale.rst

```diff
@@ -4,110 +4,258 @@
 
 .. image:: /misc/scale_ticks_below.png
 
-This is an scale with the ticks being drawn at the top of the main line:
+Example with ticks and labels drawn *on top of* the main line:
 
 .. image:: /misc/scale_ticks_on_top.png
 
-Configure ticks
----------------
-
-Set the number of total ticks with :cpp:expr:`lv_scale_set_total_tick_count(scale, total_tick_count)`
-and then configure the major tick being every Nth ticks with :cpp:expr:`lv_scale_set_major_tick_every(scale, nth_tick)`.
-
-Labels on major ticks can be configured with :cpp:expr:`lv_scale_set_label_show(scale, show_label)`,
-set `show_label` to true if labels should be drawn, :cpp:expr:`false` to hide them.
-If instead of a numerical value in the major ticks a text is required they can be set
-with :cpp:expr:`lv_scale_set_text_src(scale, custom_labels)` using ``NULL`` as the last element,
-i.e. ``static char * custom_labels[3] = {"One", "Two", NULL}``.
-
-It's possible to have the labels automatically rotated to match the ticks (only for RADIAL scales) using
-:cpp:expr:`lv_obj_set_style_transform_rotation(scale, LV_SCALE_LABEL_ROTATE_MATCH_TICKS, LV_PART_INDICATOR);`
-Or rotated a fixed amount (on any scale type) - here for 20 degrees:
-:cpp:expr:`lv_obj_set_style_transform_rotation(scale, 20, LV_PART_INDICATOR);`
-Or both at the same time
-:cpp:expr:`lv_obj_set_style_transform_rotation(scale, LV_SCALE_LABEL_ROTATE_MATCH_TICKS + 200, LV_PART_INDICATOR);`
-Some labels of the scale might be drawn upside down (to match the tick) if the scale includes a certain angle range.
+
+Configuring ticks
+-----------------
+
+You configure the major and minor ticks of a Scale by calling 2 functions:
+
+- :cpp:expr:`lv_scale_set_total_tick_count(scale, total_tick_count)`, and
+- :cpp:expr:`lv_scale_set_major_tick_every(scale, nth_tick)`.
+
+If you want labels to be drawn with the major ticks, call
+:cpp:expr:`lv_scale_set_label_show(scale, true)`.  (Pass ``false`` to hide them again.)
+
+By default, the labels shown are the numeric values of the scale at the major tick
+points.  Can you specify different label content by calling
+:cpp:expr:`lv_scale_set_text_src(scale, custom_labels)` where ``custom_labels`` is an
+array of string pointers.  Example:
+
+.. code-block:: c
+
+    static char * custom_labels[3] = {"One", "Two", NULL};
+
+Note that a ``NULL`` pointer is used to terminate the list.
+
+The content of the buffers pointed to need to remain valid for the life of the Scale.
+
+For a Scale in one of the ``..._ROUND_...`` modes, you can optionally get it to
+rotate the major-tick labels to match the rotation of the major ticks using
+:cpp:expr:`lv_obj_set_style_transform_rotation(scale, LV_SCALE_LABEL_ROTATE_MATCH_TICKS, LV_PART_INDICATOR)`.
+
+Alternately, labels can be rotated by a fixed amount (for any Scale mode).  This
+example rotates labels by 20 degrees:
+:cpp:expr:`lv_obj_set_style_transform_rotation(scale, 20, LV_PART_INDICATOR)`.
```

Comment:
I just noticed that the old docs should say that `200` is 20 degrees. This example expression should use `200`.

Reply:
I concur!  Done!